cranelift-codegen/meta/src/isa/x86/encodings.rs

```diff
@@ -1746,10 +1746,12 @@ pub(crate) fn define(
         } else {
             let template = rec_frurm.opcodes(&MOVD_LOAD_XMM);
             if ty.lane_bits() < 64 {
-                // no 32-bit encodings for 64-bit widths
                 e.enc32(instruction.clone(), template.clone());
+                e.enc_x86_64(instruction, template);
+            } else {
+                // no 32-bit encodings for 64-bit widths
```

Comment:
nit: please capitalize the first letter and end the sentence with a dot.

Can this also assert that the size if 64 in this case?